src/objectExplorer/objectExplorerService.ts

```diff
@@ -235,10 +235,16 @@ export class ObjectExplorerService {
                     this._rootTreeNodeArray.push(node);
                 });
                 this._objectExplorerProvider.objectExplorerExists = true;
-                return this._rootTreeNodeArray;
+                const sortedNodeArray = this._rootTreeNodeArray.sort((a, b) => {
```

Comment:
is it possible to refactor this method into a some smaller more easily digestible pieces?  A good rule of thumb is that methods should be less than ~20 lines of code.

also, could you please add a test for this scenario?